docs/code-block-tests.md

```diff
@@ -1,5 +1,11 @@
+---
+title: "Code Block Tests"
```

Comment:
Dont they have to have a category in here? Or not?

Reply:
Apparently they do not? Who knew! 🤷‍♂️